src/containers/layout/navbar/NavBar.jsx

```diff
@@ -22,19 +22,20 @@ import { student, tutor } from '~/constants'
 import { styles } from '~/containers/layout/navbar/NavBar.styles'
 
 const Navbar = () => {
-  const [navigationItems, setNavigationItems] = useState(
-    Object.values(guestRoutes.navBar)
-  )
   const { userRole } = useSelector((state) => state.appMain)
   const { openDrawer, closeDrawer, isOpen } = useDrawer()
   const { t } = useTranslation()
 
-  useEffect(() => {
-    if (userRole === student)
-      setNavigationItems(Object.values(studentRoutes.navBar))
-    else if (userRole === tutor)
-      setNavigationItems(Object.values(tutorRoutes.navBar))
-    else setNavigationItems(Object.values(guestRoutes.navBar))
+  const navigationItems = useMemo(() => {
+    if (userRole === student) return Object.values(studentRoutes.navBar)
+    else if (userRole === tutor) return Object.values(tutorRoutes.navBar)
+    else return Object.values(guestRoutes.navBar)
```

Comment:
in the last return we can omit using else keyword

Reply:
fixed